build_image.py

```diff
@@ -10,7 +10,7 @@
 import yaml
 
 DEFAULT_PYTHON_IMAGE = (
-    "python:3.8-bullseye@sha256:e1cd369204123e89646f8c001db830eddfe3e381bd5c837df00141be3bd754cb"
+    "python:3.9-bookworm@sha256:a23efa04a7f7a881151fe5d473770588ef639c08fd5f0dcc6987dbe13705c829"
```

Comment:
This change should be noted somewhere (probably in ofrak CHANGELOG)

Reply:
Updated all the relevant changelogs